pages/common/pio-lib.md

```diff
@@ -0,0 +1,36 @@
+# pio lib
+
+> Manage PlatformIO libraries.
+> More information: <https://docs.platformio.org/en/latest/core/userguide/lib/>.
+
+- List installed libraries:
+
+`pio lib list`
+
+- List built-in libraries based on installed development platforms and their frameworks:
+
+`pio lib buintin`
+
+- Search for existing libraries:
+
+`pio lib search {{library}}`
```

Comment:
Is this search by exact name? or keywords?

Reply:
It's using keywords. How about if I change it to `{{keyword}}`?

Reply:
@marchersimon go for it!

Reply:
Done (https://github.com/tldr-pages/tldr/pull/5728/commits/d8174eaa040d856ee04ff4bdde747e0e6f307a3a)